### ISSUE TITLE
A generated voronoi layer with role="graphics-symbol" is missing aria-label and fails a11y test

### ISSUE THREAD

**Issue by sfc-gh-yalu** [vega/vega-lite#9623]
## Bug Description

When adding a hover layer via `select` and `nearest: true`, an svg layer named `layer_1_voronoi` is generated with `role="graphics-symbol"`:

<img width="500" alt="Image" src="https://github.com/user-attachments/assets/125224a6-9ee3-4db2-85a0-f8542ee122e8" />

This is failing the a11y test (https://dequeuniversity.com/rules/axe/4.10/svg-img-alt?application=axeAPI) because it does not have an `aria-label` on it.

<img width="500" alt="Image" src="https://github.com/user-attachments/assets/70cd6b70-ac76-44eb-80b9-71a1ded21be6" />

## Code

Editor link: [https://vega.github.io/editor/#/url/vega-lite/N4IgJAzgxgFgpgWwIY...](https://vega.github.io/editor/#/url/vega-lite/N4IgJAzgxgFgpgWwIYgFwhgF0wBwqgegIDc4BzJAOjIEtMYBXAI0poHsDp5kTykBaADZ04JAGyUAVhDYA7EABoQAEzjQATjRyZ289AHE2bMoLgByCAAIImNlADWlnJqhxLbUuss6EcSopUkTBRUUAZ1QTRA4M5bBwhKKAhiEABfJUx1JFkIADM2dQQ0AG1QXJpBTDh1KOUghgRKCABPBCY2QQBebrN9AHk+-TM0gF0lOFkoNmUaWTI0UAAPBZByuEFlWqC4AMxmnB30KoQcAqRI9JBmlbWNqOcaV139w5AARwZsnWCdUjTLwRIZrVEqgZDqexRYSyHbpMFICFRdQMUwBCZTGZzFbXUKXNg4JBQOg40BTWQzHRyFbEc4MQ4ABkoAFYlASskV0DAPCDxic9mhcucIHBLjTBHS0PTUpc2UgEBBQSBZHLXlzPAFhaYoJgVnsDvc2LMdUoYQi1DrUJk6UoqehTkbqtyakooKYEQaHeo2AxjSB0dNZmQFahiiBliNpRGI0A)

Corresponding spec:
```
{
  "$schema": "https://vega.github.io/schema/vega-lite/v6.json",
  "description": "Google's stock price over time.",
  "data": {"url": "data/stocks.csv"},
  "transform": [{"filter": "datum.symbol==='GOOG'"}],
  "encoding": {
    "x": {"field": "date", "type": "temporal"},
    "y": {"field": "price", "type": "quantitative"}
  },
  "layer": [
    {  "mark": "line"},
    {
      "mark": "rule",
      "encoding": {
        "y": {},
        "opacity": {
          "condition": { "value": 0.5, "param": "hover", "empty": false},
          "value": 0
        }
      },
      "params": [
        {
          "name": "hover",
          "select": {
            "type": "point",
            "nearest": true,     // <--- This adds the `<g role="graphics-symbol" ...>` and fails a11y test
            "on": "pointerover",
            "clear": "pointerout",
            "encodings": ["x"]
          }
        }
      ]
    }
  ]
}
```

**Comment by domoritz** [comment]
Good catch. Can you send a pull request?

**Comment by sfc-gh-yalu** [comment]
I can try but can you share some pointers? The closest I found was https://github.com/vega/vega-lite/blob/main/src/compile/selection/nearest.ts but I'm not sure where the rendering logic is to add such attributes.

**Comment by domoritz** [comment]
The rendering isn't happening in Vega-Lite but in Vega. Vega-Lite just generates Vega. So start by checking what other Vega we need to generate and then go back from there.

**Comment by sfc-gh-yalu** [comment]
Thanks for the tips! Did some investigation and sent out https://github.com/vega/vega-lite/pull/9624. Let me know what you think.
